src/server.c

```diff
@@ -6487,14 +6491,12 @@ sds genRedisInfoString(const char *section) {
                 "aof_base_size:%lld\r\n"
                 "aof_pending_rewrite:%d\r\n"
                 "aof_buffer_length:%zu\r\n"
-                "aof_rewrite_buffer_length:%lu\r\n"
```

Comment:
i wonder if we want to keep this field and just transmit 0 (backwards compatibility concerns)?
let's discuss...
p.s. if we don't (or even if we do), let's list this in the top comment's "interface changes section".

Reply:
Indeed, maybe some users' monitoring systems rely on this info field, so I keep it and force it to 0.

Reply:
@yossigo WDYT?

Reply:
@oranagra @yossigo Do you have any suggestions here?

Reply:
i concluded that it's ok to just strip down this info field, updated the PR top comment.

Reply:
ohh, i now see that since i wrote down this comment we've re-introduced that field and it's always set to 0 (and the top comment already indicates that).
did we decide that in some other comment?
@yossigo WDYT? keep a dead info field always set to 0, or trim it? (i'm leaning towards trimming it)

Reply:
I think we can trim it. `INFO` is already free to omit fields depending on settings, so I guess clients should be used to not finding specific fields in some cases.

Reply:
ok.. in this case it's a field that's unlikely to be used by clients (apps), just monitoring software.
@chenyang8094 please trim it (again), and update the top comment.

Reply:
OK